apps/test/unit/templates/teacherDashboard/EmptySectionsTest.jsx

```diff
@@ -0,0 +1,63 @@
+import {render, screen} from '@testing-library/react';
+import React from 'react';
+import {BrowserRouter as Router} from 'react-router-dom';
+
+import EmptySection from '@cdo/apps/templates/teacherDashboard/EmptySection';
+import i18n from '@cdo/locale';
+
+import {expect} from '../../../util/reconfiguredChai';
+
+describe('EmptySection', () => {
+  it('displays empty desk image and Add Students link when there are no students', () => {
+    render(
+      <Router>
+        <EmptySection hasStudents={false} hasCurriculumAssigned={false} />
+      </Router>
+    );
+
+    expect(screen.getByAltText('empty desk')).to.be.visible;
+    expect(screen.getByText(i18n.addStudents())).to.be.visible;
+    expect(screen.queryByAltText('blank screen')).not.be.visible;
+    expect(screen.queryByText(i18n.browseCurriculum())).not.be.visible;
+  });
+
+  it('displays blank screen image and Browse Curriculum button when there are students but no curriculum assigned', () => {
+    render(
+      <Router>
+        <EmptySection hasStudents={true} hasCurriculumAssigned={false} />
+      </Router>
+    );
+
+    expect(screen.getByAltText('blank screen')).to.be.visible;
+    expect(screen.getByText(i18n.browseCurriculum())).to.be.visible;
+    expect(screen.queryByAltText('empty desk')).not.be.visible;
+    expect(screen.queryByText(i18n.addStudents())).not.be.visible;
+  });
+
+  it('displays empty desk image and Add Students button when there are no students but there is a curriculum assigned', () => {
+    render(
+      <Router>
+        <EmptySection hasStudents={false} hasCurriculumAssigned={true} />
+      </Router>
+    );
+
+    expect(screen.getByAltText('empty desk')).to.be.visible;
+    expect(screen.getByText(i18n.addStudents())).to.be.visible;
+    expect(screen.queryByAltText('blank screen')).not.be.visible;
+    expect(screen.queryByText(i18n.browseCurriculum())).not.be.visible;
+  });
+
+  it('only displays headline and description when there are students and a curriculum is assigned', () => {
```

Comment:
Notably this component should never be rendered if there are students in the section and the curriculum is assigned... I don't know if this means I should delete this test, modify `EmptySection` so it returns null in this case, or... do something else.  Thoughts?